mipengine/common/common_data_elements.py

```diff
@@ -23,9 +23,12 @@ class MetadataEnumeration:
 @dataclass
 class MetadataVariable:
     code: str
+    label: str
     sql_type: str
     isCategorical: bool
```

Comment:
Either use camel case or snake case but not both.

Reply:
This is coming from the data catalogue. When we integrate with them it will be much easier to change this.